experiments/axon_geometry/notebooks/resample_trace.ipynb

```diff
@@ -0,0 +1,187 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  },
+  "orig_nbformat": 4,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.6  ('env': venv)"
+  },
+  "interpreter": {
+   "hash": "c48f87b51193a18c36f6ec1b39da40df380a4c3fb2fa54b3e413dc2378ec9052"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "In order to run this notebook, you must have generated data with random dropout of trace vertices. This can be done by uncommenting the bottom of `generate_trace_data.ipynb`"
```

Comment:
```suggestion
    "In order to run this notebook, you must have generated data with random dropout of trace vertices. This can be done by uncommenting the bottom of `generate_trace_data.py`"
```